feconf.py

```diff
@@ -505,6 +506,11 @@ def get_empty_ratings():
 
 # Unfinished features.
 SHOW_TRAINABLE_UNRESOLVED_ANSWERS = False
+# Number of unresolved answers to be displayed in the dashboard for each
+# exploration.
+TOP_UNRESOLVED_ANSWERS_COUNT_DASHBOARD = 3
```

Comment:
@seanlip  I just realized that this implementation was already a part of the Version 2.3.2 released on 8th August, so it might be even live! Looking at the part of the code for calculating the count of unresolved answers per exploration - the same method is being used: https://github.com/oppia/oppia/pull/2101/files#diff-eb73e4e06cbd9d6da9674f8310fb8004R70 . Thus I guess we might have to fix this earlier than we previously intended!